deepspeed/runtime/zero/stage3.py

```diff
@@ -2027,7 +2027,7 @@ def step(self, closure=None):
             return
 
         norm_groups = self._get_norm_groups()
-        scaled_global_grad_norm = get_global_norm(norm_list=norm_groups)
+        scaled_global_grad_norm = torch.norm(torch.stack(norm_groups))
```

Comment:
I just noticed that `torch.norm` is deprecated. Is there any reason not to use the recommended replacements? 
![image](https://github.com/microsoft/DeepSpeed/assets/4271600/7e58ad0e-9d55-401b-9722-29e3586ae0f4)

Reply:
torch.linalg.norm is fine.

Reply:
@Anhelor @tjruwase Done